sdp-utils.c

```diff
@@ -1462,6 +1476,24 @@ janus_sdp *janus_sdp_generate_answer(janus_sdp *offer, ...) {
 				temp = temp->next;
 				continue;
 			}
+			if(!strcasecmp(codec, "multiopus") && (audio_fmtp == NULL ||
+					strstr(audio_fmtp, "channel_mapping") == NULL)) {
+				/* Missing channel mapping for the multiopus m-line, check the offer */
+				GList *mo = m->attributes;
+				while(mo) {
+					janus_sdp_attribute *a = (janus_sdp_attribute *)mo->data;
+					if(a->name && strstr(a->name, "fmtp") && a->value) {
+						char *tmp = strchr(a->value, ' ');
+						if(tmp && strlen(tmp) > 1) {
+							tmp++;
+							custom_audio_fmtp = g_strdup(tmp);
```

Comment:
What about (probably invalid) SDPs with multiple `fmtp` lines? In this case, I think `custom_audio_fmtp` will leak.

Reply:
Good point, thanks, I'll think whether adding a `g_free` before the `g_strdup` to get rid of the previous value, or to show a warning that we're ignoring following values instead.

Reply:
Fixed by just entering the `if` when `custom_audio_fmtp` is NULL.